crm_lead_project/models/project.py

```diff
@@ -37,3 +37,10 @@ def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
             root.set('create', 'false')
             res['arch'] = etree.tostring(root)
         return res
+
+
+class ProjectTask(models.Model):
+
+    _inherit = 'project.task'
+
+    crm_project = fields.Boolean(related='project_id.crm_project')
```

Comment:
readonly

Reply:
stored if you use it for searches but in this case you can use `project_id.crm_project` in domain and skip this new field